modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -237,6 +236,10 @@ export default {
       const isEnterPressed = e.key === 'Enter' || e.code === 'Enter' || e.code === 'NumpadEnter';
       const isSpaceBarPressed = e.keyCode === 32 || e.code === 'Space';
 
+      if (isSpaceBarPressed) {
+        e.preventDefault();
+      }
```

Comment:
Prevents scoll jump

Reply:
no side-effects? (when typing `space` in a field for instance)

Reply:
No, is only listening on the uploader UI.
tested
- Uploading a new media item and editing its details in the right-side schema
- Opening the media lib and editing an existing image's details
- Using the search bar

Use of space key was not escaped